aws/src/integration/java/org/apache/iceberg/aws/dynamodb/TestDynamoDbCatalog.java

```diff
@@ -295,6 +298,37 @@ public void testDropNamespace() {
     Assert.assertFalse("namespace must not exist", response.hasItem());
   }
 
+  @Test
+  public void testRegisterTable() {
```

Comment:
This pair of tests is repeated (in a very similar way) across multiple catalogs. Can those be centralized somewhere? `CatalogTests` maybe?